lib/dogapi/v1/comment.rb

```diff
@@ -9,36 +9,24 @@ class CommentService < Dogapi::APIService
 
       # Submit a comment.
       def comment(message, options = {})
-        begin
-          body = {
-            'message' => message,
-          }.merge options
-
-          request(Net::HTTP::Post, "/api/#{API_VERSION}/comments", true, nil, body, true)
-        rescue Exception => e
-          suppress_error_if_silent e
-        end
+        body = {
+          'message' => message,
+        }.merge options
+
+        request(Net::HTTP::Post, "/api/#{API_VERSION}/comments", true, nil, body, true)
       end
 
       # Update a comment.
       def update_comment(comment_id, options = {})
-        begin
-          if options.empty?
-            raise "Must update something."
-          end
-
-          request(Net::HTTP::Put, "/api/#{API_VERSION}/comments/#{comment_id}", true, nil, options, true)
-        rescue Exception => e
-          suppress_error_if_silent e
+        if options.empty?
+          raise "Must update something."
```

Comment:
Technically changes the behavior here as this exception won't be rescued anymore